INSTALL.md

```diff
@@ -0,0 +1,67 @@
+# How to install MacGitver
+
+In present state, you need to compile MacGitver from source. We provide a compilable version from our default branch `development`. If you like to help with creating binary packages, please create a [pull request](https://github.com/macgitver/MacGitver/pulls) or contact me via [mail](mailto:nils@macgitver.org).
+
+## Prerequisites
```

Comment:
Indeed, but when writing that the only supported way of installation is to build via source, you obviously cannot leave that fact out. A better solution might be then to just add the info to the prerequisite section: "Qt >= 5.4 via Qt Installer"